tests/lib/rule-tester/flat-rule-tester.js

```diff
@@ -1116,6 +1116,56 @@ describe("FlatRuleTester", () => {
         }());
     });
 
+    it("should allow setting the filename to a non-JavaScript file", () => {
+        ruleTester.run("", require("../../fixtures/testers/rule-tester/no-test-filename"), {
+            valid: [
+                {
+                    code: "var foo = 'bar'",
+                    filename: "somefile.ts"
+                }
+            ],
+            invalid: []
+        });
+    });
+
+    it("should allow setting the filename to a file name without extension", () => {
+        ruleTester.run("", require("../../fixtures/testers/rule-tester/no-test-filename"), {
+            valid: [
+                {
+                    code: "var foo = 'bar'",
+                    filename: "somefile"
+                }
+            ],
+            invalid: []
+        });
+    });
+
+    it("should allow setting the filename to a file path without extension", () => {
+        ruleTester.run("", require("../../fixtures/testers/rule-tester/no-test-filename"), {
+            valid: [
+                {
+                    code: "var foo = 'bar'",
+                    filename: "path/to/somefile.js"
```

Comment:
This file path has an extension so the description seems wrong? While at this, it might be best to have two tests: `path/to/somefile.js` and `path/to/somefile`.

Reply:
I had the pattern already set to `"**"` and did not double check that I modified the test case. Anyways fixed now.